src/vm/amd64/cgencpu.h

```diff
@@ -225,6 +215,18 @@ void UpdateRegDisplayFromCalleeSavedRegisters(REGDISPLAY * pRD, CalleeSavedRegis
 
 #define NUM_ARGUMENT_REGISTERS 4
 
+#define ENUM_CALLEE_SAVED_REGISTERS() \
+    CALLEE_SAVED_REGISTER(Rdi) \
+    CALLEE_SAVED_REGISTER(Rsi) \
+    CALLEE_SAVED_REGISTER(Rbx) \
```

Comment:
Is there a reason why the order of registers is different between Windows (..., rbx, rbp, r12, r13, r14, r15) and Unix (r12, r13, r14, r15, rbx, rbp)?